addons/website_slides/models/slide_slide.py

```diff
@@ -287,6 +311,8 @@ def create(self, values):
                 values.setdefault(key, value)
 
         slide = super(Slide, self).create(values)
+        if values.get('completion_time'):
```

Comment:
Unnecessary for a computed field.

Another way would be to do it in an onchange instead of a computed field and allow values from front-end.